components/server/src/billing/billing-mode.ts

```diff
@@ -33,7 +32,7 @@ export class BillingModes {
         return { mode: "usage-based", paid };
     }
 
-    async getBillingModeForUser(user: User, now: Date): Promise<BillingMode> {
+    async getBillingModeForUser(): Promise<BillingMode> {
```

Comment:
`getBillingModeForUser` without any user context looks pretty odd from the outside. Do we need this method at all?

Reply:
I added a comment, and explicitly marked it as deprecated now. All we really need to do is add orgnaizationId to `api.maySetTimeout` and `api.updateWorkspaceTimeoutSetting`, then we can remove it entirely.